src/main/java/com/vitalii/twenty/methods/TwentyMethods.java

```diff
@@ -24,4 +24,14 @@ public static int returnMaxOfTwoInts(int a, int b) {
         }
         return b;
     }
+
+    //3.1 приминает 2 инта, а и б, возвращает большее из этих 2х чисел
+    public static int returnMaxOfTwoIntsWithMath(int c, int d) {
+        return Math.max(c, d);
+    }
+
+    //3.2 приминает 2 инта, а и б, возвращает большее из этих 2х чисел
+    public static int returnMaxOfTwoIntsWithThernary(int a, int b) {
+        return (a > b) ? a : b;
```

Comment:
Use more informative names of variables in all functions in class:
int a -> int firstNumber
int b -> int secondNumber

Reply:
to be fixed in next pull request